activate_interpreter.sh

```
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Активация интерпретатора для проекта BMI-App_2025_clean
+# Использование: source activate_interpreter.sh
+
+echo "🚀 Активация интерпретатора для BMI-App_2025_clean..."
+
+# Переходим в директорию проекта
+cd /Users/katsiaryna_kavaleuskaya/Documents/BMI-App_2025_clean
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)